nilearn/decoding/space_net.py

```diff
@@ -1134,6 +1129,26 @@ def _binarize_y(self, y):
         self.n_classes_ = len(self.classes_)
         return y
 
+    def score(self, X, y):
+        """Returns the mean accuracy on the given test data and labels.
+
+        Parameters
+        ----------
+        X : list of Niimg-like objects
+            See http://nilearn.github.io/manipulating_visualizing/manipulating_images.html#niimg
+            Data on which model is to be fitted. If this is a list,
+            the affine is considered the same for all.
+
+        y : array or list of length n_samples.
+            Labels.
+
+        Returns
+        -------
+        score : float
+            Mean accuracy of self.predict(X)  w.r.t y.
```

Comment:
People that are not familiar with convex optimization might not know what "w.r.t." could mean.